File: SEP3-TIER2/src/Server/Model/ChannelUser.java
package Server.Model;

public class ChannelUser {

    private int channelid;
    private String username;

    public ChannelUser() {
    }

    public int getChannelid() {
        return channelid;
    }

    public void setChannelid(int channelid) {
        this.channelid = channelid;
    }

    public String getUsername() {
        return username;
    }

    public void setUsername(String username) {
        this.username = username;
    }

    @Override
    public String toString() {
        return "ChannelUser{" +
                "channelid=" + channelid +
                ", username='" + username + '\'' +
                '}';
    }
}

File: SEP3-TIER1/BlazorTest/Model/ChannelUser.cs
using Newtonsoft.Json;

namespace BlazorTest.Model
{
    public class ChannelUser
    {
        [JsonProperty("channelid", NullValueHandling = NullValueHandling.Ignore)]
        public int ChannelId { get; set; }

        [JsonProperty("username", NullValueHandling = NullValueHandling.Ignore)]
        public string Username { get; set; }
    }
}
